app/main.py

```diff
@@ -2,0 +1,14 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+    sourse_file = parts[1]
+    dest_file = parts[2]
+    if sourse_file == dest_file:
+        return
+    try:
+        with (open(sourse_file, "r") as file_in,
+              open(dest_file, "w") as file_out):
```

Comment:
Consider adding error handling for other exceptions that might occur during file operations, such as `PermissionError` or `IOError`, to make the function more robust.